staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/validation/validation_test.go

```diff
@@ -282,6 +288,11 @@ func TestValidateMangedFieldsValid(t *testing.T) {
 			Operation:  metav1.ManagedFieldsOperationApply,
 			FieldsType: "FieldsV1",
 		},
+		{
+			Operation:  metav1.ManagedFieldsOperationApply,
+			FieldsType: "FieldsV1",
+			Manager:    "🍔",
```

Comment:
I hope you don't mind be borrowing that burger here, just to test the fieldManager code gets called.